src/components/CreateResetPassword/components/ChangePasswordConfirmation/ChangePasswordConfirmation.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import {
+  Button,
+  Headline,
+} from '@folio/stripes-components';
+
+import OrganizationLogo from '../../../OrganizationLogo';
+
+import styles from './ChangePasswordConfirmation.css';
+
+const ChangePasswordConfirmation = ({ history }) => {
```

Comment:
I do not see where do you pass history to this component? Is this `withRouter` behavior?

Reply:
Yes, withRouter is a HOC passing history object in props to the wrapped component

Reply:
Yes, it comes in withRouter HOC